### PR TITLE
Migrate from indexer client and use postgraphile for fetching data from PostgreSQL

### PR DESCRIPTION
**Pull request by rmeena840**
Migrate from indexer client and use postgraphile for fetching data from PostgreSQL
Task: https://www.pivotaltracker.com/story/show/182318157

### CODE REVIEW

File: src/pages/AllDao.vue [hunk 1]
```diff
@@ -362,7 +362,7 @@ export default defineComponent({
 							key: index,
 							dao_id: +item.appId,
 							token_id: +item.assetId,
-							token_name: tokenData.name as string,
+							token_name: tokenData.an as string,
```
Comment (Megha-Dev-19) [inline src/pages/AllDao.vue:365]: Why is it `an`

File: src/pages/Proposal.vue [hunk 2]
```diff
@@ -245,9 +245,9 @@ export default defineComponent({
 						}
 					}
 
-					// check if asset is already opted
+					// check if app is already opted
 					const isApplicationAlreadyOpted = await isApplicationOpted(
-						this.walletStore.address,
+						lsig.address(),
```
Comment (Megha-Dev-19) [inline src/pages/Proposal.vue:250]: Why are we using lsig here, shouldn't it be specific to user connected, to check if he/she has opted the app

File: src/utility/algod.utility.ts [hunk 3]
```diff
@@ -91,3 +91,8 @@ export const optInDaoApp = async (
 		throw error;
 	}
 };
+
+export const convertToHex = (address: string) => {
+	const pk = algosdk.decodeAddress(address);
+	return Buffer.from(pk.publicKey).toString("hex");
+};
```
Comment (Megha-Dev-19) [inline src/utility/algod.utility.ts:98]: I didn't understand the use of this function, what does it do.?

File: src/pages/AllDao.vue (hunk 1 above)
Comment (rmeena840) [reply src/pages/AllDao.vue:365]: We are now directly using the PostreSQL for fetching data. Indexer Client used to do these proper naming before it returns the response on client side but now we are fetching directly. This is how it is stored in PostreSQL.

File: src/utility/algod.utility.ts (hunk 3 above)
Comment (rmeena840) [reply src/utility/algod.utility.ts:98]: It converts the address into a hexadecimal address. The address is stored in PostgreSQL in hexadecimal encoding.

File: src/pages/Proposal.vue (hunk 2 above)
Comment (rmeena840) [reply src/pages/Proposal.vue:250]: I have checked the add propsal script. It checks if it has been opted by lsig or not. So lsig will be here not the wallet address.

`// check if app is already opted
					const isApplicationAlreadyOpted = await isApplicationOpted(
						lsig.address(),
						this.daoStore.dao_id as number
					);
					// optin
					if (!isApplicationAlreadyOpted) {
						await this.optInLsigToApp(lsig);
					}`

check here we are trying to opt in lsig

File: src/pages/AllDao.vue (hunk 1 above)
Comment (Megha-Dev-19) [reply src/pages/AllDao.vue:365]: okay, thanks for the information

File: src/utility/algod.utility.ts (hunk 3 above)
Comment (Megha-Dev-19) [reply src/utility/algod.utility.ts:98]: okay, I see that PostgreSQL stores data quite differently than indexer

File: src/pages/Proposal.vue (hunk 2 above)
Comment (Megha-Dev-19) [reply src/pages/Proposal.vue:250]: okay

**Review by Megha-Dev-19 — APPROVED** [review]
Lgtm

File: src/pages/AllDao.vue (hunk 1 above)
Comment (robert-zaremba) [inline src/pages/AllDao.vue:365]: let's try to avoid using `as`

**Pull request merged**